src/components/common/bottomSheet/BottomSheet.tsx

```diff
@@ -0,0 +1,158 @@
+import styled from "@emotion/styled";
+import theme from "../../../styles/theme";
+import { commonIcons } from "../../../assets/icons";
+import { keyframes } from "@emotion/react";
+import { useState } from "react";
+
+const fadeIn = keyframes`
+  from { opacity: 0; }
+  to   { opacity: 1; }
+`;
+
+const fadeOut = keyframes`
+  from { opacity: 1; }
+  to   { opacity: 0; }
+`;
+
+const slideUp = keyframes`
+  from { transform: translateX(-50%) translateY(100%); }
+  to   { transform: translateX(-50%) translateY(0); }
+`;
+
+const slideDown = keyframes`
+  from { transform: translateX(-50%) translateY(0); }
+  to   { transform: translateX(-50%) translateY(100%); }
+`;
+
+export interface Option {
+  value: string;
+  label: string;
+}
+
+interface BottomSheetProps {
+  title?: string; //바텀시트 제목
+  options: Option[]; //옵션 목록
+  selected: string; //선택된 옵션 value값
+  onSelect: (value: string) => void; //옵션 선택할 때 콜백함수
+  onClose: () => void; //오버레이, 버튼 클릭 시 호출
+}
+
+const BottomSheet: React.FC<BottomSheetProps> = ({
+  title = "정렬순서",
+  options,
+  selected,
+  onSelect,
+  onClose,
+}) => {
+  const { close: Close } = commonIcons;
+  const [isClosing, setIsClosing] = useState(false);
```

Comment:
명시적으로 초기화를 false로 잘 해주셨지만, ts를 쓰는만큼 
```suggestion
  const [isClosing, setIsClosing] = useState<boolean>(false);
```
으로 명시하면 더 좋을 거 같아요!

Reply:
수정했습니다 !